modules/data.atmosphere/R/download.AmerifluxLBL.R

```diff
@@ -92,7 +95,11 @@ download.AmerifluxLBL <- function(sitename, outfolder, start_date, end_date,
   
   if (download_file_flag) {
     extract_file_flag <- TRUE
-    utils::download.file(ftplink, output_zip_file)
```

Comment:
We can just always use `PEcAn.utils::download.file(ftplink, output_zip_file, method=method)` then in there put the logic to switch, that way we can use the same code everywhere.